HexDump.Benchmark/FormatBenchmark.cs

```diff
@@ -0,0 +1,65 @@
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+
+namespace HexDump.Benchmark
+{
+    // [ClrJob, CoreJob, MonoJob]
+    //[SimpleJob(RuntimeMoniker.Net472, baseline: true)]
+    //[SimpleJob(RuntimeMoniker.NetCoreApp30)]
+    [SimpleJob(RuntimeMoniker.NetCoreApp31)]
+    //[SimpleJob(RuntimeMoniker.CoreRt30)]
+    //[SimpleJob(RuntimeMoniker.Mono)]
+    [RPlotExporter]
```

Comment:
Consider adding [`MemoryDiagnoser`](https://benchmarkdotnet.org/articles/configs/diagnosers.html) here too.

Reply:
Yep, that's good idea !

Reply:
good idea !